doc/en/configuration_guide/imp/index.rst

```diff
@@ -184,7 +185,11 @@ Don't forget that each pack use a plugin to work. Each monitoring procedure will
 install the plugin needed.
 
 .. note::
-    Install plugins on each poller required. Otherwise your supervision will not work.
+    Install plugins on each poller required. Otherwise your supervision will
```

Comment:
Install plugins on each poller required -> Installing plugins on each poller is required

Reply:
corrected